### PR TITLE
Assign BEP044 to "Stimuli BEP"

### PR DESCRIPTION
**Pull request by yarikoptic**
Attn @neuromechanist  @dorahermes @VisLab @smakeig

### CODE REVIEW

File: _data/beps.yml [hunk 1]
```diff
@@ -403,3 +403,23 @@
...
   pull_request_created:
   pull_request_merged:
+
+- number: "044"
+  title: Stimuli
+  leads:
+    - name: "[Seyed Yahya Shirazi](mailto:shirazi@ieee.org)"
+    - name: Dora Hermes
+    - name: Yaroslav O. Halchenko
+    - name: Tal Pal Attia
```
Comment (dorahermes) [inline _data/beps.yml:413]: Tal has left science

Comment (yarikoptic) [reply _data/beps.yml:413]:
```suggestion
```

**Comment by Remi-Gau** [comment]
will probably have to open a similar PR against main

**Comment by yarikoptic** [comment]
> will probably have to open a similar PR against main

oh, it isn't automagically somehow populated from this metadata?

**Comment by Remi-Gau** [comment]
Na. The main branch was created from a completely different repo. So changes on the old website have to be mirrored on the new one.
Hence why we have been avoiding changing the old one unless 100% necessary.

**Review by neuromechanist — APPROVED** [review]
LGTM

**Pull request merged**